ios/RNEngageBySailthru.m

```diff
@@ -102,7 +106,7 @@ - (EngageBySailthru *)engageBySailthruWithResolver:(RCTPromiseResolveBlock)resol
         }
     }
 
-    [[self engageBySailthruWithResolver:resolve rejecter:reject] setAttributes:marAttributes withResponse:^(NSError * _Nullable error) {
+    [[self engageBySailthruWithRejecter:reject] setAttributes:marAttributes withResponse:^(NSError * _Nullable error) {
```

Comment:
You already have the instance from the call above, you shouldn't need to call it again